Competition/src/main/java/competition/subsystems/climber/commands/HoldRobotLevelCommand.java

```diff
@@ -0,0 +1,45 @@
+package competition.subsystems.climber.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.climber.RearMotorClimberSubsystem;
+import competition.subsystems.pose.PoseSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDFactory;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.PropertyFactory;
+
+public class HoldRobotLevelCommand extends BaseCommand {
+
+    RearMotorClimberSubsystem rear;
+    PoseSubsystem pose;
+    PIDManager pitchPid;
+    PIDManager rollPid;
+
+    @Inject
+    public HoldRobotLevelCommand(RearMotorClimberSubsystem rear, PoseSubsystem pose, PropertyFactory propFactory, PIDFactory pf) {
+        this.rear = rear;
+        this.pose = pose;
+
+        pitchPid = pf.createPIDManager(getPrefix() + "PitchControl", 0.05, 0, 0, 1, -0.2);
+        rollPid = pf.createPIDManager(getPrefix() + "RollControl", 0.05, 0, 0, 1, -0.2);
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+        rear.calibrateFloor();
```

Comment:
thinking about this more, I think this is too dangerous, it means you can't quickly flip to manual and back to hold level without totally messing up the height calibration. Propose floor calibration is done as a separate, explicit command for now.